specs/source/management/identities.yml

```diff
@@ -355,6 +355,35 @@ paths:
           $ref: '../shared/standard-responses.yml#/responses/notFoundResponse'
         '401':
           $ref: '../shared/standard-responses.yml#/responses/unauthorizedResponse'
+  identities-id-trace:
+    parameters:
+      - $ref: '../shared/parameters.yml#/id'
+    put:
```

Comment:
Is there a reason PUT was chosen over POST?

PUT is usually associated with "create with id" and "update with id". This feels like a POST to create a new data entry. The odd part is that you can't read the current details of the trace (i.e. no GET).

We have this same pattern with MFA supporting POST and GET.

Reply:
Mmmm I can wrangle my brain into PUT being okay. Since there is only one record and no id necessary.

Reply:
It's kind of a weird middle case. I don't have a strong preference. If you like POST better, I can switch it. I thought of it more as attaching some data to identity, rather than a standalone entity, but you can see it either way.